manferna/01-basics/01-basics-output-string.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: output-string
+spec:
+  
+  steps:
+    - name: output-string
+      image: registry.access.redhat.com/ubi9@sha256:4227a4b5013999a412196237c62e40d778d09cdc751720a66ff3701fbe5a4a9d
+      env:
+        - name: FILE_PATH
+          value: $(workspaces.shared-workspace.path)/input-file.txt
+      script: |
+        #!/usr/bin/env bash
+        cat ${FILE_PATH}
+      
+  workspaces:
```

Comment:
You can make the workspace readOnly for this task as you only read from it. I think it's mounted for each task, so you can mount it once RW for the write task and one RO for the read task.